tests/e2e/features/dicomViewer.feature

```diff
@@ -8,8 +8,8 @@ Feature: Preview dicom image
     Given the user "Admin" has logged in
     And the user has uploaded the dicom file "MRBRAIN.dcm"
     When the user previews the dicom file "MRBRAIN.dcm"
-    And the user checks VIP metadata for dicom file
-    Then the user should see patient name "MR/BRAIN/GRASE/1024" in VIP metadata
+    Then the user should see the dicom file "MRBRAIN.dcm"
+    And the user should see patient name "MR/BRAIN/GRASE/1024" in VIP metadata
     When the user checks the extended metadata for dicom file
     Then the user should see patient name "MR/BRAIN/GRASE/1024" in metadata sidebar
     And the user closes the DICOM file preview
```

Comment:
the last line is not really a Then step. If we want to test if closing works properly, then line 15 would be a When statement and I would need to be followed by a Then step for the assertion. 

Since closing the file is a standard functionality of ocis web, I don't think we need to test this in this scenario. However closing the sidebar is something you might want to add to the scenario.